src/generate/formats/writer/yaml/index.ts

```diff
@@ -443,8 +445,14 @@ function writeConfigPages(project: Project, integration: IntegrationObjectFromYA
                 } else {
                   includes.configVar = true;
                   writer
-                    .writeLine(`"${escapeDoubleQuotes(configVar.name)}": configVar({`)
-                    .writeLine(`stableKey: "${camelCase(configVar.key)}",`)
+                    .writeLine(`"${escapeText(configVar.name)}": configVar({`)
+                    .writeLine(
+                      `stableKey: "${
+                        configVar.dataType === "htmlElement"
+                          ? camelCase(configVar.key).substring(0, MAX_HTML_STABLEKEY_LENGTH)
```

Comment:
Should we consider doing a hash on the substring to make the `stableKey` more opaque (less evident it's a substring of the value at export)? I'm a bit concerned end-users may think they need to update the stableKey when they update the value as it stands (also I'm a little surprised HTML elements have stable keys since we don't need to carry values over).

Reply:
HTML elements need stable keys to be reimported as CNI, otherwise users will run into type errors.

Doing a hash seems fine, I'll throw that in there.